estate/util.py

```diff
@@ -0,0 +1,6 @@
+def action(function):
+    """When nothing is returned, return `True` as a default instead"""
+    def action_wrapper(*args, **kwargs):
+        if function(*args, **kwargs) == None:
+            return True
+    return action_wrapper
```

Comment:
A little bit overkill in this context but still pretty cool to use custom decorators